Program.cs

```diff
@@ -3,7 +3,13 @@
 for (; ; ) {
    Console.Write ("Enter the word:");
    string word = Console.ReadLine ().ToLower ();
+   if (!Isogram (word))
+      Console.WriteLine (word + " is not an isogram.");
+   else Console.WriteLine (word + " is an isogram.");
+}
+bool Isogram (string word) {

```

Comment:
Add the empty line between the functions.
You can write the function in one line like
`bool Isogram (string word) => word.Distinct ().Count () == word.Length;`